pkgs/edge-worker/src/EdgeWorker.ts

```
@@ -90,14 +90,12 @@ export class EdgeWorker {
     config: EdgeWorkerConfig = {}
   ): Promise<PlatformAdapter> {
     if (typeof handlerOrFlow === 'function') {
-      console.log('start() if typeof handlerOrFlow is function');
-      return this.startQueueWorker(
+      return await this.startQueueWorker(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)